easybuild/easyconfigs/a/ABINIT/ABINIT-7.10.5-intel-2016.02-GCC-4.9.eb

```diff
@@ -0,0 +1,35 @@
+# initially generated with eb ABINIT-7.10.4-intel-2015a.eb --try-software-version=7.10.5
+# --try-toolchain=intel,2016.02-GCC-4.9, then added intel mpi and fft support
+easyblock = 'ConfigureMake'
+
+name = 'ABINIT'
+version = "7.10.5"
+
+homepage = 'http://www.abinit.org/'
+description = """ABINIT is a package whose main program allows one to find the total energy,
+ charge density and  electronic structure of systems made of electrons and nuclei (molecules
+ and periodic solids) within Density Functional  Theory (DFT), using pseudopotentials and a
+ planewave or wavelet basis."""
+
+toolchain = {'name': 'intel', 'version': '2016.02-GCC-4.9'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['http://ftp.abinit.org/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = []
+
+# ensure mpi and intel toolchain
+configopts = '--with-fc-vendor=intel --with-linalg-flavor=mkl --enable-mpi ' \
+             '--with-fft-flavor=fftw3-mkl ' \
+             '--with-fft-libs="-lmkl_intel_lp64 -lmkl_sequential -lmkl_core"'
+
+# there are variants (e.g. --with-dft-flavor), but is an easyconfig file needed for all?
+# eb --try-amend can be used to install those variants from the same easyconfig file...
```

Comment:
d1ac97c added an easyconfig file for a variant --with-dft-flavor=libxc. Added "_libxc" as versionsuffix and named the file according to the convention, I think, but isn't it confusing to have the (software) version suffix after the toolchain? it looks like a toolchain suffix, maybe I missed something.